pages/documentation/migration-V3.md

```diff
@@ -650,33 +633,30 @@ $theme-utility-breakpoints: (
     ========================================
     ========================================
     */
-
     // -------------------------------------
     // Import individual theme settings
-
     @forward "uswds-theme"; // or whatever your theme file is named
-
     ...
     ```
 
-    Now your project is using its theme settings in the proper USWDS 3.0 format!
+Now your project is using its theme settings in the proper USWDS 3.0 format!
 
 #### Use "uswds-core" for any custom USWDS Sass
 
 {:.border-top-2px.border-base-lighter.padding-top-1}
-Sass Module syntax requires that files must note the source of any tokens, variables, mixins, functions, or placeholders used in that file. 
+Unlike `@import`, which makes Sass members (tokens, variables, mixins, functions, or placeholders) available globally, `@use` only reveals Sass members to the stylesheet that loads them. 
```

Comment:
Introduced the term `members` in the way it is used in the [Sass Docs](https://sass-lang.com/documentation/at-rules/use#loading-members) so that we wouldn't have to repeat this list throughout this section.